### ISSUE TITLE
Some questions about paper

### ISSUE THREAD

**Issue by WangZixuan** [jackd/template_ffd#8]
How do you guarantee that the deformed mesh is still manifold(keeps the topolopy)?
In training, if we do not restrict the range that the control points of FFD vary, the mesh will be no more manifold. Do we have to consider this and if yes, where is the corresponding code?

**Comment by jackd** [comment]
We don't. The meshes don't begin manifold, and we do nothing to explicitly encourage manifold-ness. This makes the meshes of limited value over the point clouds - and we tossed up whether or not to even say we have a mesh output. I proposed claiming to produce 'arbitrarily dense point clouds', but somebody questioned what the difference was between a mesh and an arbitrarily dense point cloud and I didn't have a good answer, so we went with the more common-place term.

If you want a manifold I'd look into reconstructing manifolds from point clouds. Note that uniformly sampling the template will result in non-uniform sampling of the output cloud (unlike deforming the vertices and sampling the resulting mesh uniformly), so using some basic denoising techniques could introduce different topologies as regions of low point density get culled.
